### PR TITLE
set truck width & height properly

### PR DESCRIPTION
**Pull request by nilsnolde**
fixes #31

it's not the prettiest but gets the job done;)

### CODE REVIEW

File: src/Controls/Directions/index.jsx [hunk 1]
```diff
@@ -30,6 +31,7 @@ class DirectionsControl extends React.Component {
   handleUpdateProfile = (event, data) => {
     const { dispatch } = this.props
     dispatch(updateProfile({ profile: data.valhalla_profile }))
+    dispatch(resetSettings())
```
Comment (nilsnolde) [inline src/Controls/Directions/index.jsx:34]: reset settings when profile updates

File: src/Controls/settings-options.js [hunk 2]
```diff
@@ -520,6 +520,12 @@ export const settingsInit = {
   use_geocoding: true,
 }
 
+export const settingsInitTruckOverride = {
+  ...settingsInit,
+  width: 2.6,
+  height: 4.11,
+}
+
```
Comment (nilsnolde) [inline src/Controls/settings-options.js:528]: urgh I know, but wasn't feeling like more of a rewrite

File: src/Controls/index.jsx [hunk 3]
```diff
@@ -124,6 +125,7 @@ class MainControl extends React.Component {
         }
       })
       dispatch(zoomTo(processedCoords))
+      dispatch(resetSettings())
```
Comment (nilsnolde) [inline src/Controls/index.jsx:128]: this will reset when it's loaded from a URL

**Review by chrstnbwnkl — APPROVED** [review]
fine for me, at this point it's gotta be either a dirty quick fix or an entire refactoring, nothing in between :smile:

**Comment by nilsnolde** [comment]
agreed!

**Pull request merged**